bucket/raine.json

```
@@ -1,80 +1,80 @@
-                    "RAINE"
-                ]
-            ],
-            "shortcuts": [
-                [
-                    "raine32.exe",
-                    "RAINE"
-                ]
-            ]
-        },
-        "64bit": {
-            "url": [
-                "https://raine.1emulation.com/archive/raine64-0.94.12b.7z",
-                "https://raine.1emulation.com/archive/dlls64-0.92x.7z"
-            ],
-            "hash": [
-                "0d51f70cecdf37cb8e50e6753ed9462cb464e7c3125900cbe0d11d95413373df",
-                "0b312986da5acd806a0a247f5f8a6ba601cbfbb1e3671518b53d4fd5eb6754a4"
-            ],
-            "extract_dir": "raine64",
-            "bin": [
-                [
-                    "raine.exe",
-                    "RAINE"
-                ]
-            ],
-            "shortcuts": [
-                [
-                    "raine.exe",
-                    "RAINE"
-                ]
-            ]
-        }
-    },
-    "persist": [
-        "artwork",
-        "blend",
-        "config",
-        "debug",
-        "demos",
-        "emudx",
-        "roms",
-        "savedata",
-        "savegame",
-        "screens"
-    ],
-    "checkver": {
-        "url": "https://raine.1emulation.com/download/latest.html",
-        "regex": "Available files for version ([\\d.]+[a-zA-Z]?)"
-    },
-    "autoupdate": {
-        "architecture": {
-            "64bit": {
-                "url": "https://raine.1emulation.com/archive/raine64-$version.7z"
-            },
+	"version": "0.95",
+	"description": "RAINE is an emulator for arcade games",
+	"homepage": "https://raine.1emulation.com/",
+	"license": "Artistic-2.0",
+	"architecture": {
+		"32bit": {
+			"url": [
+				"https://raine.1emulation.com/archive/raine32-0.95.7z",
+				"https://raine.1emulation.com/archive/dlls32-0.95.7z"
+			],
+			"hash": [
+				"bdac0049f15bb4db2485fa488fcddeaa6b4baec4cfe7035bab6568e3381efe77",
+				"6623542ffc992d7a481c9f44dcb9a39a265712ba6407b0a3680ff85caceb1f0b"
+			],
+			"extract_dir": "raine32",
+			"bin": [
+				[
+					"raine32.exe",
+					"RAINE"
+				]
+			],
+			"shortcuts": [
+				[
+					"raine32.exe",
+					"RAINE"
+				]
+			]
+		},
+		"64bit": {
+			"url": [
+				"https://raine.1emulation.com/archive/raine64-0.95.7z",
+				"https://raine.1emulation.com/archive/dlls64-0.95.7z"
+			],
+			"hash": [
+				"4318eedd75dce831fecefe045a080fdd9189dd660f258d6ae1fa02be8bdf35fd",
+				"c5078df325870e325993c8d8d16dd16e1c2b94ef6f9467e0e41c9b7b367b4828"
+			],
+			"extract_dir": "raine64",
+			"bin": [
+				[
+					"raine.exe",
+					"RAINE"
+				]
+			],
+			"shortcuts": [
+				[
+					"raine.exe",
+					"RAINE"
+				]
+			]
+		}
+	},
+	"persist": [
+		"artwork",
+		"blend",
+		"config",
+		"debug",
+		"demos",
+		"emudx",
+		"roms",
+		"savedata",
+		"savegame",
+		"screens"
+	],
+	"checkver": {
+		"url": "https://raine.1emulation.com/download/latest.html",
+		"regex": "Available files for version ([\\d.]+[a-zA-Z]?)"
```

checkver is reading `0.95.0` from the site, while the binaries use `0.95`
```suggestion
		"regex": "/archive/raine32-([\\d.]+[a-zA-Z]?)\\.7z"
```